deleteDiscordMessages.js

```diff
@@ -50,19 +53,35 @@
     const startBtn = popup.document.querySelector('button#start');
     const stopBtn = popup.document.querySelector('button#stop');
     const autoScroll = popup.document.querySelector('#autoScroll');
+    const fileSelection = popup.document.querySelector("input#file");
+    fileSelection.addEventListener("change", () => {
+        const files = fileSelection.files;
+        const channelIdField = popup.document.querySelector('input#channelId');
+        if (files.length > 0) {
+            const file = files[0];
+            file.text().then(text => {
+                let json = JSON.parse(text);
+                let channels = []
+                Object.keys(json).forEach(k => {
+                    channels.push(k);
+                })
+                channelIdField.value = channels.join(";");
```

Comment:
```suggestion
                channelIdField.value = channels.join(", ");
```
I think separating things with comma is more natural for non technical ppl